org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -0,0 +1,187 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
+/* jshint sub:true */
+// This code uses ES2015 syntax that requires at least Node.js v4.
+// For Node.js ES2015 support details, reference http://node.green/
+
+"use strict";
+var request = require('request-promise');
+
+/**
+* This translator class implements the "Hub" interface.
+*/
+class Translator {
+    constructor(accessToken) {
+        this._accessToken = accessToken;
+
+        this._baseUrl = "https://api.wink.com";
+        this._devicesPath = '/users/me/wink_devices';
+
+        this._name = "Wink Hub"; // TODO: Can be pulled from OpenT2T global constants. This information is not available, at least, on wink hub.
+    }
+
+    /**
+     * Get the list of devices discovered through the hub.
+     */
+    getDevicesAsync() {
```

Comment:
Yes the Async suffix is supported (but optional) for property getters/setters and any methods. But as mentioned above, this should be refactored to OCF style methods.

---
In reply to: [76154581](https://github.com/openT2T/translators/pull/75#discussion_r76154581) [](ancestors = 76154581)